.gitignore

```diff
@@ -1,4 +1,5 @@
 # custom
+cluster.yaml
```

Comment:
shouldn't we move that to be in the config folder. Related to that I don't think it should be called config as this might conflict with the user project structure. `.thunderbolt` or `.thunderbolt-config` @paselem and @jiata what do you think?

Reply:
yeah, this is a mistake. The cluster.yaml and secrets.cfg should be moved into the config/ directory. `.thunderbolt-conf` is probably a better name as well.

Reply:
@jiata seems to prefer `.thunderbolt`